src/GameState/Marble.tsx

```diff
@@ -1,12 +1,25 @@
-import { Circle, Pressable, Box } from 'native-base';
+import { Circle, Pressable, Box, View } from 'native-base';
 import React from 'react';
+import { ImageBackground } from 'react-native';
 
-const MARBLE_COLOR_RADIAL = ['warmGray.300', 'black', 'white', 'red.700'];
+const RED_MARBLE = require('./red-marble.png');
```

Comment:
It's ok, but you could have use an svg, and change the color, resulting in loading less ressources. But again, it's ok